packages/nx/src/command-line/add/add.ts

```diff
@@ -0,0 +1,131 @@
+import { exec } from 'child_process';
+import * as ora from 'ora';
+import { isAngularPluginInstalled } from '../../adapter/angular-json';
+import type { GeneratorsJsonEntry } from '../../config/misc-interfaces';
+import { logger } from '../../utils/logger';
+import { output } from '../../utils/output';
+import {
+  getPackageManagerCommand,
+  type PackageManagerCommands,
+} from '../../utils/package-manager';
+import { getPluginCapabilities } from '../../utils/plugins';
+import { workspaceRoot } from '../../utils/workspace-root';
+import type { AddOptions } from './command-object';
+
+export async function addHandler(args: AddOptions): Promise<void> {
+  output.addNewline();
```

Comment:
Should we wrap this in `handleErrors` for consistency with several other command handlers? https://github.com/nrwl/nx/blob/4a5659ef7fa283e49ef2980ec8fca7d47aaa98c1/packages/nx/src/utils/params.ts

Reply:
Having now read the rest you will need to verify the behaviour of your wrapping of `exec` below if we use handleErrors

Reply:
Yeah I meant more around the user-facing logging might be inconsistent when manually exiting the process _sometimes_

Reply:
It might not be an issue in practice but worth playing with it